### ISSUE TITLE
Terminal addon accessibility problems

### ISSUE THREAD

**Issue by afr-ux** [home-assistant/frontend#24682]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [x] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

The official terminal add-on is not accessible with screen readers.
There are 2 problems:
1- The edit box for entering text. When you write in this editing box, the text you have written is not spoken and cannot be edited, as it is not spoken.
2- The result window is not accessible to screen readers. It is completely invisible to screen readers.
Tested with JAWS and NVDA in chrome, firefox and edge in Windows 10. This problem has been with homeassistant for at least a year.
Proposed solution: use a standard control in both situations. In addition, in the result window, aria-live should be used, in a similar way to a chat window, so that when new text appears, it is automatically read by screen readers.

### Describe the behavior you expected

Read information with screenreader succesfully

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

All

### In which browser are you experiencing the issue?

chrome, firefox, edge

### Which operating system are you using to run this browser?

Windows 10 Pro

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### Javascript errors shown in your browser console/inspector

```txt

```

### Additional information

The official terminal add-on is not accessible with screen readers.
There are 2 problems:
1- The edit box for entering text. When you write in this editing box, the text you have written is not spoken and cannot be edited, as it is not spoken.
2- The result window is not accessible to screen readers. It is completely invisible to screen readers.
Tested with JAWS and NVDA in chrome, firefox and edge in Windows 10. This problem has been with homeassistant for at least a year.
Proposed solution: use a standard control in both situations. In addition, in the result window, aria-live should be used, in a similar way to a chat window, so that when new text appears, it is automatically read by screen readers.

**Comment by karwosts** [comment]
I suspect this might be more accurately reported to the addons repo.

https://github.com/home-assistant/addons/issues

**Comment by afr-ux** [comment]
Please, if any admin considers that this issue should be better placed in another category, please move it, as I am not able to find that option with my screenreader. Thank you.
